docs/about/overview/radio-settings.mdx

```diff
@@ -1,67 +1,97 @@
 ---
 id: radio-settings
 title: Radio Settings
+slug: /overview/radio-settings
 sidebar_label: Radio Settings
-sidebar_position: 3
+sidebar_position: 1
 ---
 
-We use the same channel maps as LoRaWAN (though this is not LoRaWAN).
-(Note, not the same channel map as TTN for US frequencies.)
+:::info
+Meshtastic is **not** LoRaWAN, nor TTN (TheThingsNetwork). However, Meshtastic does use the same channel maps as LoRaWAN.
+:::
 
 ![freq table](/img/LoRa-Frequency-Bands.jpg)
 
 See [this site](https://www.rfwireless-world.com/Tutorials/LoRa-channels-list.html) for more information.
 
-## LoRaWAN Europe Frequency Band
+## Europe Frequency Bands
 
-The maximum power allowed is +14dBm ERP (Effective Radiated Power, see [this site](https://en.wikipedia.org/wiki/Effective_radiated_power) for more information).
+The maximum power allowed for Europe is +14dBm ERP ([Effective Radiated Power](https://en.wikipedia.org/wiki/Effective_radiated_power)).
```

Comment:
+14dBm true for 433 only. EU_868 can use up to +27dBm ERP ina very narrow sub-band we are restricting ourselves to.
The limits will generally be lifted in the software if is_licensed is set (HAM Mode)